python/src/aac/plugins/check/check_aac_impl.py

```diff
@@ -17,7 +17,17 @@
 
 
 def check(aac_file: str, fail_on_warn: bool, verbose: bool) -> ExecutionResult:  # noqa: C901
-    """Business logic for the check command."""
+    """Business logic for the check command.
+
+    Args:
+        aac_file (str):         The AaC file being processed
+        fail_on_warn (bool):    Flag to fail when warnings are discovered
+        verbose (bool):         Flag for verbose mode. When true add success messages as encountered.
+
+    Returns:
+        ExecutionResult:        Method result containing: plugin_name ("Check AaC"), "check", status, message
+                                including results from lower level helper methods
+    """
```

Comment:
Have LanguageError for check method as a whole as well that needs captured in a Raises entry.

```suggestion
    Returns:
        ExecutionResult:        Method result containing: plugin_name ("Check AaC"), "check", status, message
                                including results from lower level helper methods
                                
     Raises:
         LanguageError: When a defining schema for the requested root could not be found. 
    """
```